src/__stories__/SchemaRow.tsx

```diff
@@ -0,0 +1,82 @@
+import { Provider as MosaicProvider } from '@stoplight/mosaic';
+import { withKnobs } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import { JSONSchema4 } from 'json-schema';
+import * as React from 'react';
+
+import { SchemaRow } from '../components';
+import { buildTree, findNodeWithPath } from '../components/shared/__tests__/utils';
+import { Wrapper } from './utils/Wrapper';
+
+const schema: JSONSchema4 = require('../__fixtures__/schemaRowBugs/schemaRowBugs.json');
+
+let tree = buildTree(schema);
+
+storiesOf('Bugs|SchemaRow Component.oneOf', module)
```

Comment:
@P0lip : I created these bugs in Storybook two-fold: 
1. So that others had a way of seeing the bugs in action
2. So that I could debug them.

It was expected that these would be taken out of Storybook after the bug fix. Instead of yalc'ing, is there another method of debugging you can recommend (other than using Storybook)?

Reply:
> So that others had a way of seeing the bugs in action
> It was expected that these would be taken out of Storybook after the bug fix. I

gotcha, sounds good :+1:

> Instead of yalc'ing, is there another method of debugging you can recommend (other than using Storybook)?

You could write a test for that, we have quite a few of them, and then run debugger from there or perform a console based debugging.